src/models/thelio-major-r3/repairs.md

```diff
@@ -0,0 +1,483 @@
+
+## Replacing the CPU cooler and CPU:
+
+The CPU cooler dissipates heat from the CPU to the heatsink, where the CPU fans expel it from the system. Depending on your climate and the age of the machine, replacing the thermal paste between the CPU and the cooler/heatsink may help the system run cooler.
+
+The CPU uses an AMD AM5 socket. The CPU cooler is a [Noctua NH-U12S](https://noctua.at/en/nh-u12s/specification).
+
+**Tools required:** Cross-head (Phillips) screwdriver, thermal paste  
+**Time estimate:** 45 minutes  
+**Difficulty:** High <span style="color:red;">●</span>
+
+### Steps to remove the CPU cooler/thermal paste:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the GPU brace and GPU](#replacing-the-gpu), and [remove the CPU shroud](#removing-the-cpu-shroud).
+2. While holding the CPU cooler in place so it doesn't fall, unscrew the two screws holding the CPU cooler crossbar onto the vertical mounting brackets.
+    - The screws are held captive, and will not fully come out of the crossbar.
+    - Only the two outer screws need to be removed; the center screw holding the horizontal crossbar onto the heatsink does not need to be removed.
+
+![CPU cooler screws](./img/cpu-cooler-screws.webp)
+
+3. The cooler will come away from the CPU.
+4. Using a paper towel, clean the existing thermal paste off of the heatsink and CPU. You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+### Steps to replace the CPU:
+
+1. Place the computer on its side so the motherboard is facing up.
+2. Unscrew and remove the cooler mounting bracket on the right side of the CPU.
+
+![CPU cooler mounting bracket](./img/cpu-cooler-mounting-bracket.webp)
+
+3. Push the locking pin outward until it's able to spring away from the motherboard.
+    - Caution: the locking pin may spring up with significant force when freed.
+
+![CPU locking pin](./img/cpu-locking-pin.webp)
+
+4. Flip the CPU holder away from the CPU.
+    - The CPU holder opens in the opposite direction from the locking pin.
+
+![CPU holder raised](./img/cpu-holder-raised.webp)
+
+5. Carefully lift the CPU out of the CPU socket.
+    - Be careful not to bend any of the gold pins on the CPU socket, and do not touch the gold pads on the CPU.
+
+![CPU (removed)](./img/cpu-removed.webp)
+
+6. Gently place the new CPU into the socket.
+    - The triangle on the CPU should be oriented to match the triangle on the CPU cover, pointing towards the top left of the motherboard.
+7. Flip the CPU cover back onto the CPU and push the locking pin down into place.
+8. Reinstall the right-side cooler mounting bracket.
+
+### Steps to install the thermal paste/CPU cooler:
+
+1. Draw an `X` shape of thermal paste onto the CPU.
+
+![Thermal paste application](./img/thermal-paste-application.webp)
```

Comment:
It's too bad the right side cooler mount isn't installed in this picture. Reinstalling it is mentioned in step 8 above though, so I think this image is probably okay.

Reply:
Ugh, I agree it should be there since that step was listed above.

Reply:
Ah well, minor detail. We can fix it later if anyone gets confused by it.

Reply:
I disagreed on the compromise, so fixed in https://github.com/system76/tech-docs/pull/162/commits/ee490964fef0222dd33bd8f52c2c8660f6b420bd.

Reply:
Nice, that looks great!